bfv/params.go

```diff
@@ -13,24 +13,15 @@ import (
 // MaxLogN is the log2 of the largest supported polynomial modulus degree.
 const MaxLogN = 16
 
+// MinLogN is the log2 of the smallest supported polynomial modulus degree.
+const MinLogN = 4
```

Comment:
Why not 1 ? Also it should be explicit that this is the minimum supported log2 for correctness of the operations, but that such small instances are insecure.

Reply:
Minimum must be 2^4 to ensure the NTT correctness.

Reply:
Ok for me. Mention this in the comment too.

Reply:
Done